# Data and experiment folders
workspace: "/home/oskar/icetrack/workspace/long3"
exp: "test"

# Calibration files
int_file: "/home/oskar/smooth_sailing/src/smooth_sailing/cfg/calib/int_right.yaml"
ext_file: "/home/oskar/smooth_sailing/src/smooth_sailing/cfg/calib/ext_right.yaml"

# Input topics
imu_topic: "/arduino_node/imu"
gnss_topic: "/arduino_node/gnss"
lidar_topic: "/livox_lidar_node/pointcloud2"
image_topic: "/blackfly_node/image"

# Output topics
pose_topic: "/icetrack/pose_stamped"
lidar_frame_topic: "/icetrack/lidar_frame"
raw_cloud_topic: "/icetrack/raw_cloud"

### Clean up this later ###
max_playback_rate: -1